core/src/main/java/kafka/server/share/SharePartition.java

```diff
@@ -1326,6 +1355,42 @@ int leaderEpoch() {
         return leaderEpoch;
     }
 
+    // Visible for testing
+    void recordFetchLockRatioMetric(long acquiredTime) {
```

Comment:
Javadoc comment here please, in particular for the parameter.

Reply:
Done.